root/static/scripts/edit/MB/Control/ArtistCredit.js

```diff
@@ -76,7 +76,9 @@ MB.Control.ArtistCreditName = aclass(MB.entity.ArtistCreditName, {
 
         } else if (/^&$/.test(join)) {
             this.joinPhrase(" & ");
-        }
+        }else if(/^;$/.test(join) ){
```

Comment:
Yeah, I think I was in CB mode and didn't pay mind to the `{}`s. I need to do fewer things at the same time…